aries-site/src/pages/components/skeleton.mdx

```diff
@@ -15,16 +15,19 @@ import { DashBoardSkeletonExample, SkeletonAnatomy } from '../../examples';
   <DashBoardSkeletonExample />
 </Example>
 
+### Guidance
+Page skeletons are used in scenarios where the content is being fetched asynchronously and may take some time to load. A skeleton's anatomy is based on the UI element it is rendering and should be responsive.
+
 ### When to use
-Skeletons are used on all pages to provide a visual layout of the content being loaded.  They help users perceive a loading period as being shorter in duration.
+- The interface is content-rich, with numerous elements such as images, text, and other media that take time to load and are known.
+- The content is fetched asynchronously, especially from different sources or APIs. Using a skeleton can maintain the structure of the page and prevent layout shifts when content starts loading in multiple regions.
+- The expected response time is greater than ~300ms.
 
 ### When not to use
-Skeletons should not be used if:
-- The data or elements being loaded are unknown.
-- The user is awaiting feedback for in-context actions such as submitting a form or fetching search results.
-  - The Spinner component may be more appropriate.
-- The loading process takes more than a few minutes to load.
-  - Pair with a message or notification to provide the user with better visibility of a system’s status.
+- The data or elements being loaded are dynamic. Use a [Spinner] (/components/spinner) instead.
```

Comment:
I believe skeleton would also apply when the data is dynamic, so I'd remove this section. I'd think we would always know what ultimately would render in a section of a page, because if the data was there, we'd render it.

Reply:
This is a discussion I was having with @KennyAtHPE... He mentioned that the main difference between skeleton and spinner is that skeleton is only for known content and spinner is for unknown. We changed the word to dynamic because unknow was hard to understand. Do you have a better way to put it? or are you saying that this distinction should not be made?

Reply:
I'm more suggesting the distinction should not be made. Since skeleton is aiming to give a scaffold of the page as data is being fetched, I don't think there's really a scenario when elements are unknown. Happy to discuss this further next week on a call if helpful!

Reply:
I will schedule some time for us